### PR TITLE
[core] Fixed some compiler warnings.

### PR DESCRIPTION
**Pull request by maxsharabayko**
- C4101 - local variable never used (2).
- C4129- 'character' : unrecognized character escape sequence (1).
- C4305 - truncation from 'type1' to 'type2' (1).

Related to #1646

### CODE REVIEW

File: srtcore/congctl.cpp [hunk 1]
```diff
@@ -498,7 +498,7 @@ class FileCC : public SrtCongestionControlBase
         HLOGC(cclog.Debug, log << "FileCC: LOSS: "
             << "sent=" << CSeqNo::seqlen(m_iLastAck, m_parent->sndSeqNo()) << ", inFlight=" << pktsInFlight
             << ", lost=" << numPktsLost << " ("
-            << lost_pcent_x10 / 10 << "." << lost_pcent_x10 % 10 << "\%)");
+            << lost_pcent_x10 / 10 << "." << lost_pcent_x10 % 10 << "%%)");
```
Comment (ethouris) [inline srtcore/congctl.cpp:501]: Are you sure that this shouldn't be in parentheses?
```
<< (lost_pcent_x10 / 10) << "." << (lost_pcent_x10 % 10) << "%)");
```

BTW is this double % needed? This facility doesn't use a printf-format.

**Review by ethouris — COMMENTED** [review]
Consider  ^^

File: srtcore/congctl.cpp (hunk 1 above)
Comment (maxsharabayko) [reply srtcore/congctl.cpp:501]: Hm, indeed...

**Comment by ethouris** [comment]
One more thing: core.cpp line 11187 - `pending_broken` should be under `ENABLE_EXPERIMENTAL_BONDING`

**Comment by maxsharabayko** [comment]
> One more thing: core.cpp line 11187 - `pending_broken` should be under `ENABLE_EXPERIMENTAL_BONDING`

Fixed.

**Pull request merged**